src/PIL/TgaImagePlugin.py

```diff
@@ -118,8 +118,9 @@ def _open(self) -> None:
             start, size, mapdepth = i16(s, 3), i16(s, 5), s[7]
             if mapdepth == 16:
                 self.palette = ImagePalette.raw(
-                    "BGR;15", b"\0" * 2 * start + self.fp.read(2 * size)
+                    "ARGB;1555", b"\0" * 2 * start + self.fp.read(2 * size)
                 )
+                self.palette.mode = "RGBA"
```

Comment:
```suggestion
```
Without #7966, this line makes the palette mode RGBA only until the image is loaded, after which it becomes RGB.

Removing the line makes it consistently RGB. I don't know if it's better to consistently wrong or inconsistently wrong though.

Reply:
If #7966 is merged I can rebase this on that. Otherwise if this is merged first this change would need to be made in #7966.

Reply:
I think I meant the opposite what you're saying - if #7966 is merged, this line is helpful. If #7966 is not merged, then I think this line should be removed.

Reply:
#7966 is now merged.

Reply:
Thanks

Reply:
This TGA change actually isn't quite complete. I've left a comment in https://github.com/python-pillow/Pillow/pull/7965

Reply:
Since #7966 has been merged, and this has been rebased on that, this change is no longer necessary. However, it was noticed in #7965 that the TGA alpha channel is actually inverted, so "ARGB;1555" isn't quite correct here.

Reply:
https://github.com/python-pillow/Pillow/pull/7965 has now been merged.